### DISCUSSION TITLE
Copying a git directory

### DISCUSSION THREAD

**Discussion by lamyseba** [git-lfs/git-lfs#4532] [Q&A] [answered]
Hi, I copied a directory of a project, which was tracked with git & git lfs. Copy occured over disks with different file system (ntfs -> f2fs) Cloning the original is no more an option as it is already erased.
All my files stored with git lfs are marked as modified when I run `git status` or `git lfs status`.

As a hint, git lfs status gives me output  of this kind, for files marked as modified: 

> static/home.jpg (LFS: d7ccdfb -> File: d7ccdfb)

In this example, it seems that the file `/static/home.jpg` has the same "print", but marked as "LFS" in the repository, and as "File" in my working copy.

I don't know what to do in order to have git LFS recognizing files that are not modified. I don't want to store a new version of the file with the same file, this would waste a lot of disk space. Thanks for your help

**Comment by lamyseba** [comment, marked as answer]
Hi, answering my own dumb question: running
```
cd path/to/the/copied/directory
git lfs install
```
 solved the problem (replace path/to/the/copied/directory with your own path to your project root directory)

**Reply by bk2204** [reply]
I was just about to suggest the same thing.  If `git lfs install` hasn't been run on your system, then this will likely cause files that used to be stored in LFS to be converted to Git files.  In my case, I've committed the settings set by `git lfs install` into my `.gitconfig`, so as soon as I set up a new machine and deploy my dotfiles, Git LFS is set up and I don't need to run `git lfs install`.

The reason all those files suddenly got modified is that when you moved file systems, the Git index information became stale and Git re-read all of those files, and since Git LFS wasn't installed, they got turned into Git files instead of LFS files.

Anyway, glad you figured it out.

**Reply by lamyseba** [reply]
The trick is that git lfs needs a twoo step install: first I installed it as a package on my new system. After installing the package, I was not aware I needed to run this command on my repositories.
Thanks for the trick  of the `.gitconfig` file, that will remove this second step.
